tools/cldr-unittest/src/org/unicode/cldr/unittest/TestCLDRFile.java

```diff
@@ -844,4 +846,50 @@ public void TestExtraPaths() {
         assertTrue("Should be empty", missing.isEmpty());
     }
 
+    public void testNewCache () {
+
+        CLDRFileCache.USE_CLDRFILE_CACHE = true;
+        String[] locales = {"root", "en", "en_GB", "zh", "zh_Hant", "zh_Hans_SG", "zh_Hans_HK"};
+        Map<String, CLDRFile> mapNew = new HashMap<>();
+        Map<String, CLDRFile> mapNewR = new HashMap<>();
+        Map<String, CLDRFile> mapOld = new HashMap<>();
+        Map<String, CLDRFile> mapOldR = new HashMap<>();
+
+
+        for(String locale: locales) {
```

Comment:
package org.unicode.cldr.util;

import com.ibm.icu.text.DecimalFormat;
import com.ibm.icu.text.NumberFormat;
import com.ibm.icu.util.ULocale;

public final class Timer {

Reply:
Call in a loop, where you get a file multiple times.